app/src/main/java/net/grandcentrix/blutufintegration/ui/LaunchFragment.kt

```diff
@@ -69,7 +72,7 @@ class LaunchFragment : Fragment() {
 
     @VisibleForTesting
     fun isBluetoothSupported(): CheckResult {
-        val result = when (BluetoothRepository.deviceSupportsBluetooth(requireContext())) {
+        val result = when (repository.deviceSupportsBluetooth(requireContext())) {
```

Comment:
For this PR it is fine as it is not related. But a Fragment should be dump and so it should not access a repository directly. This is the responsibility of a `ViewModel`.
Maybe this might be a next task to work on the precondition checking and resolving.